recipes/pennylane-lightning-kokkos/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pennylane-lightning-kokkos" %}
+{% set version = "0.29.1" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PennyLaneAI/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f51ba7718defc7bb5064f690f381e04b2ec58cb09f22a171ae5f410860716e30
+  patches:
+    - 0001-setup.patch
+    - 0002-setup.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: |
+    {{ PYTHON }} setup.py build_ext --define="PLKOKKOS_ENABLE_WARNINGS=OFF"
+    {{ PYTHON }} setup.py bdist_wheel
+    {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
This package installs some unused CMake files into site-packages. Open a patch upstream to remove them for the next release, please.

Reply:
I see that. The goal is to ship the header templated library along with the binaries. I'll patch the glob expression to fix this, if that alright.

Reply:
I think you're asking if modifying `MANIFEST.in` is the correct way to exclude that file. I think it is.